content/en/service_management/workflows/private_actions.md

```diff
@@ -0,0 +1,180 @@
+further_reading:
+- link: "service_management/app_builder/connections"
+  tag: "Documentation"
+  text: "App Builder Connections"
+- link: "service_management/workflows/connections"
+  tag: "Documentation"
+  text: "Workflow Connections"
+- link: "service_management/workflows/guide/private_action_credentials"
+  tag: "Documentation"
+  text: "Handling Private Action Credentials"
+---
+
+{{< callout url="https://google.com" btn_hidden="false" header="Join the Beta!">}}
+Private Actions are in beta. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+Private actions allow your Datadog workflows and apps to interact with services hosted on your private network without exposing your services to the public internet. To use private actions, you must install a private action runner on a host in your network and pair the runner with a Datadog Connection.
+
+The following integrations support private actions:
+- [Kubernetes][1]
+- [Postgres][2]
+- [Jenkins][3]
+- [Temporal][4]
+- The [HTTP][5] action
+
+## Prerequisites
+
+The private action runner requires a Linux host that is able to reach any internal services you want to call from an action or app. To use App Builder with private actions, you must be able to point a hostname to the private action runner container. This hostname must be resolvable by your App users. App Builder calls the hostname using HTTPS, so you must bring your own SSL termination.
+
+In addition, the host has the following requirements:
+- 2GB of RAM
+- Docker (with Docker Compose if that is your preference)
+
+## Set up a private action runner
+
+From the **Private Action Runner** page in [Workflow Automation][6] or [App Builder][7], click **New Private Action Runner**. The installation steps differ depending on whether you want to install the runner for App Builder, Workflow Automation, or both App Builder and Workflow Automation.
+
+{{% collapse-content title="Both App Builder and Workflow Automation" level="p" %}}
+1. Enter a name for your runner.
+1. Click **Both**.
+1. Enter a runner hostname. App Builder calls your runner using this hostname over HTTPS. You must bring your own SSL termination and forward to port 9016 in the container.
+1. Create a directory on your host where the runner can store its configuration. For example, `./config`. You can also use this directory to [store any credentials][10] required by the runner's connection.
+1. Deploy your runner with Docker or Docker Compose:
+
+{{< tabs >}}
+{{% tab "Docker" %}}
+1. Click **Docker**.
+1. Run the provided `docker run` command on your host, replacing `./config` with the path to the directory you created for the runner configuration.
+
+   You can safely ignore the error: `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
+{{% /tab %}}
+
+{{% tab "Docker Compose" %}}
+1. Click **Docker Compose**.
+1. Create a `docker-compose.yaml` file and add the provided YAML, or add the `runner` stanza to an existing Docker Compose file.
```

Comment:
Does it matter where they do this?

Reply:
I'll add a link to the Docker Compose docs somewhere here, cos I don't want to document Docker's stuff - If a user selects this option, we should assume some knowledge of Compose though.